docs/guides/web-ui/live.md

```diff
@@ -47,72 +54,56 @@ CREATE TABLE records AS (
 )
 ```
 
-Some basic examples to get started:
-
-- To view your warnings and errors type: `level > 'info'`
-- To see just exceptions type: `is_exception`
-- To filter by service name (which you can find on the detail panel of any given trace): `service_name = 'crud-api'`
-
-## Details panel closed
-
-![Logfire OpenAI Image Generation](../../images/logfire-screenshot-live-view.png)
-
-This is what you'll see when you come to the live view of a project with some data.
+If you're not sure where to start, scroll down to the `Start here` section of the search box for some suggestions.
 
-1. **Organization and project labels:** In this example, the organization is `samuelcolvin`, and the project is `logfire-demo-spider`. You can click the organization name to go to the organization overview page; the project name is a link to this page.
+[TODO: Add image of `Start here` suggestions]
 
-2. **Project pages:** These are links to the various project-specific pages, including the Live, [Dashboards](./dashboards.md), [Alerts](./alerts.md), [Explore](./explore.md), and Settings pages.
+### Ask in English -> Get SQL
+Ask in English: Allows you to write your query in English, and will then convert that to SQL. Great if you're not
+confident with SQL and/or can't quite remember a tricky clause. You have the option to either create a completely
+new query with `Get new SQL` or (if you have some SQL already) modify a work-in-progress query with `Modify existing SQL`.
+You'll note that whenever you click either of these buttons the SQL is updated in the SQL Search box above.
 
-3. **Feedback button:** Click the feedback button to provide us feedback.
+[TODO: Add image of natural language query]
 
-4. **Light/Dark mode toggle:** Cycles between light, dark, and system — because everyone seems to have an opinion on this :smile:
+Under the hood this uses
+an LLM running with [PydanticAI](https://github.com/pydantic/pydantic-ai).
 
-5. **Link to the current view:** Clicking this copies a link to the page you are on, with the same query etc.
+### Common Clauses
+Common clauses: A list of pre-populated useful queries. A powerful way to rapidly generate the query you need, whilst
+simultaneously learning more about all the ways you can search your data. Clicking multiple clauses will add them to
+your query with an `AND` statement. If you'd like something other than an `AND` statement you can either modify the
+SQL populated in the SQL search or use the [explore screen](explore.md) for more complex queries.
 
-6. **Organization selection panel:** Opens a drawer with links to the different organizations you are a member of, and also has links to the Terms and Conditions, Support, Documentation, and a Log Out button.
```

Comment:
I've deliberately culled a lot of these - some are no longer relevant, but the section is currently just a bit too much of a "wall of text". I've tried to pull out the key/least intuitive areas to describe